### PR TITLE
Remove SlackReporter

### PR DESCRIPTION
**Pull request by lptr**
Removes an unused web hook for Slack.

### CODE REVIEW

**Comment by lptr** [comment]
@bot-gradle test QFL please

File: subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractCrossVersionGradleProfilerPerformanceTest.groovy [hunk 1]
```diff
@@ -59,12 +55,10 @@ class AbstractCrossVersionGradleProfilerPerformanceTest extends Specification {
 
     def setup() {
         def gradleProfilerReporter = new GradleProfilerReporter(temporaryFolder.testDirectory)
-        def slackReporter = reporter
-        def compositeReporter = CompositeDataReporter.of(slackReporter, gradleProfilerReporter)
         runner = new GradleProfilerCrossVersionPerformanceTestRunner(
             new GradleProfilerBuildExperimentRunner(gradleProfilerReporter.getResultCollector()),
             resultStore,
-            compositeReporter,
+            resultStore,
```
Comment (blindpirate) [inline subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractCrossVersionGradleProfilerPerformanceTest.groovy:61]: Two identical params?

**Comment by guylabs** [comment]
@lptr shouldn't this go to `release` as well such that we are sure that it's not used anymore on the main CI ? @blindpirate Please remove the environment variables `env.SLACK_PERFORMANCE_REPORT_CHANNEL` and `env.SLACK_WEBHOOK_URL` from [here](https://builds.gradle.org/admin/editProject.html?projectId=Gradle&tab=projectParams) as well when that is merged.

**Comment by lptr** [comment]
This should be cherry-picked to `release`, yes.

File: subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractCrossVersionGradleProfilerPerformanceTest.groovy (hunk 1 above)
Comment (lptr) [reply subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractCrossVersionGradleProfilerPerformanceTest.groovy:61]: We pass the same object once as a store once as a reporter.

**Comment by lptr** [comment]
Cherry-picked to `release` as e1ab35224ca1ca328011c58a61528c60a1824477.

**Pull request merged**